tasks/base/log/logHelpers.go

```diff
@@ -135,7 +135,8 @@ func collectFilePaths(envVars map[string]string, configElements []baseConfig.Val
 		paths = append(paths, "/tmp")                                     //For Python Agent log and PHP installation log
 		paths = append(paths, "/var/log")                                 //For Syn Minion and Infra
 		paths = append(paths, "/var/log/newrelic")                        // For PHP agent and daemon log
-		paths = append(paths, "/usr/local/newrelic-netcore20-agent/logs") // for dotnetcore
+		paths = append(paths, "/usr/local/newrelic-netcore20-agent/logs") // for dotnetcore up to v10
+		paths = append(paths, "/usr/local/newrelic-dotnet-agent") // for dotnetcore v10+
```

Comment:
Should this have also had `logs` on the end of the path?

Reply:
https://github.com/newrelic/newrelic-diagnostics-cli/pull/128